packages/web-client/app/helpers/config.ts

```diff
@@ -0,0 +1,18 @@
+import { getOwner } from '@ember/application';
+import Helper from '@ember/component/helper';
+import { get } from '@ember/object';
+
+export default class extends Helper {
+  config: Config | undefined;
```

Comment:
Config is pointing to qunit.Config here. The union type with undefined makes any property access invalid to Typescript, since undefined has no keys.